esp-link/config.c

```diff
@@ -35,6 +35,8 @@ FlashConfig flashDefault = {
   .data_bits	= EIGHT_BITS,
   .parity	= NONE_BITS,
   .stop_bits	= ONE_STOP_BIT,
+  .telnet_port1 = 23,
+  .telnet_port2 = 2323,
```

Comment:
Ill take a look to see how that was done in some other code and proceed appropriately.